### PR TITLE
Create anonymize method stub with flask.

### PR DESCRIPTION
**Pull request by shiranr**
Create anonymize method stub with flask

### CODE REVIEW

File: presidio-anonymizer/presidio_anonymizer/presidio_anonymizer.py [hunk 1]
```diff
@@ -1 +1,13 @@
-"""Main module."""
+from flask import Flask, request
```
Comment (omri374) [inline presidio-anonymizer/presidio_anonymizer/presidio_anonymizer.py:1]: We should decide if the flask app is part of the package (i.e. inside the `presidio_anonymizer` folder) or outside, in some `app.py` file, which is similar to how we had the grpc server set up.

**Review by omri374 — APPROVED** [review]
Previous

File: presidio-anonymizer/presidio_anonymizer/presidio_anonymizer.py (hunk 1 above)
Comment (SharonHart) [inline presidio-anonymizer/presidio_anonymizer/presidio_anonymizer.py:1]: I would start structuring the project, have this file just run the app and move the route to views.py / routes.py

**Review by SharonHart — APPROVED** [review]
Minor comment + please add a PR description

File: presidio-anonymizer/presidio_anonymizer/presidio_anonymizer.py (hunk 1 above)
Comment (shiranr) [reply presidio-anonymizer/presidio_anonymizer/presidio_anonymizer.py:1]: Not sure I understand the remark. 
The entire application is going to be inside the presidio_anonymizer folder just like in the anonymizer (I am trying to keep consistency between the applications).
If I am not mistaking @itye-msft and I are per-programming the flask on the analyzer, so if the layout will change, we will also change the anonymizer.

Comment (shiranr) [reply presidio-anonymizer/presidio_anonymizer/presidio_anonymizer.py:1]: Will be done later with the analyzer for consistency.

**Pull request merged**